docs/internals/contributing/bugs-and-features.txt

```diff
@@ -29,6 +29,11 @@ general points:
   likely to get lost. If a particular ticket is controversial, please move the
   discussion to |django-developers|.
 
+.. Note::
+
+    Django does not use GitHub for bug reports or feature requests.
```

Comment:
I'm concerned this might create confusion in light of other projects under the Django umbrella that do use for GitHub for reporting issues. I don't know if we should actually make any changes to address the ticket. I think if the reporter had simply read https://code.djangoproject.com/newticket there wouldn't have been an issue. We get reports in the wrong tracker occasionally but I'm not convinced that someone who doesn't read the stuff on the new ticket page is likely to find these notes. Any thoughts?